update A_lambda in hwc_colors.py

koheimiyakawajp opened the pull request:
updated "A_lambda ()" as Yano-san instructed,
and the old one was moved to "A_lambda_linear ()" in hwc_colors.py.

koheimiyakawajp committed:
updated the function for A_lambda.

koheimiyakawajp committed:
updated A_lambda in hwc_colors.py

astronasutarou commented:
ありがとうございます．
とりいそぎ以下のコマンドでフォーマットを整えたものを追い push します．

``` console
$ autopep8 -i --select E225,E226,E227,E231,E261,E265,E271,E302,E303,E502 src/telescope_baseline/hw_band_coeff/hwc_colors.py 
```

Commit:
format by autopep8

``` console
$ autopep8 -i --select E225,E226,E227,E231,E261,E265,E271,E302,E303,E502 src/telescope_baseline/hw_band_coeff/hwc_colors.py
```

astronasutarou commented:
ちょっと状況を整理させてください．

今回の PR でファイル `hwc_colors.py` を追加していますが，もともとの目的は `Hw_coeff.py` の関数 (`A_lambda`) を更新することだと聞いていたような気がします．特に強い理由がなければ `Hw_coeff.py` をアップデートするのが良いと思います．

koheimiyakawajp commented:
申し訳ありません。私のローカルにあるコードとごちゃまぜにしてしまっていました。。
Hw_coeff.pyの方のアップデートで対応します。

koheimiyakawajp committed:
update A_lambda()

koheimiyakawajp committed:
Merge branch 'feature/hw_fix' of https://github.com/JASMINE-Mission/telescope_baseline into feature/hw_fix

koheimiyakawajp committed:
remove hwc_color.py

koheimiyakawajp commented:
Hw_coeff.pyのアップデートで対応しました。
また、先に上げた不要ファイル(hwc_colors.py)は削除しました。
よろしくお願い致します。

astronasutarou reviewed (approved):
ありがとうございます！コードの更新を確認しました．

astronasutarou commented:
ありゃ？ conflict がありますね．

Commit:
Merge branch 'develop' into feature/hw_fix

astronasutarou commented:
online editor で conflict を解決しました

astronasutarou commented:
あれ？僕が編集したせいでテスト落ちていますかね……確認します．

astronasutarou commented:
うーむ…… `feature/Hw_fix` のルートとなっている commit がちょっと古かったのでしょうか？ `develop` から同期をかけたときにこのブランチにいろいろと降ってきましたね．

astronasutarou reviewed (changes requested):
すみません．extinction law を変えたので計算結果が変わります．計算結果と対応させているテストで引っかかるようになっているようです．新しい `A_lambda` の計算でテストが通るようにテスト内容を確認していただけないでしょうか (テストの書き方を見直すべきだとは思うのですがとりあえず今回はこれでお願いします)．

koheimiyakawajp committed:
edit Hw_coeff_test.py

koheimiyakawajp committed:
remove comments from  Hw_coeff_test.py

koheimiyakawajp commented:
テスト内容も合わせてアップデートしました。

astronasutarou reviewed (approved):
更新を確認しました．ありがとうございます！
